packages/cadl-python/test/generated/dev-driven/resiliency/devdriven/_operations/_operations.py

```diff
@@ -24,6 +25,8 @@
 from azure.core.tracing.decorator import distributed_trace
 from azure.core.utils import case_insensitive_dict
 
+from .. import models as _models
```

Comment:
With Cadl DPG models, we know the models folder strucutre will be `models/_models.py`. THerefore we can make this import `from ..models import _models` and that way we don't have to special-case the hidden models we use (i.e. paging models). We honestly can use this for legacy models as well and do `from ..models import _models_py3 as _models`

Reply:
Hi @iscai-msft , 

Would you give specific link for what we want to change for legacy models?

The only place I found we are using `_models_py3` is like here: https://github.com/Azure/autorest.python/blob/20220705-dpg-model/packages/autorest.python/test/azure/legacy/Expected/AcceptanceTests/StorageManagementClient/storage/models/__init__.py#L9.
I think we don't need to change this part?